digiwf-integrations/digiwf-s3-integration/digiwf-s3-integration-client-starter/src/main/java/io/muenchendigital/digiwf/s3/integration/client/configuration/S3IntegrationClientAutoConfiguration.java

```diff
@@ -38,22 +43,38 @@ public class S3IntegrationClientAutoConfiguration {
 
     public final S3IntegrationClientProperties s3IntegrationClientProperties;
 
-    /**
-     * Creates a bean with name "apiClientFactory" of {@link ApiClientFactory}.
-     *
-     * This factory class is providing either the preconfigured {@link FileApiApi} or {@link FileApiApi}.
-     *
-     * @param restTemplate to create rest requests.
-     *                     If the S3 integration service is secured via Oauth2,
-     *                     the OAuth2RestTemplate can be used here, for example.
-     * @return the {@link ApiClientFactory}.
-     */
     @Bean
-    public ApiClientFactory apiClientFactory(final RestTemplate restTemplate) {
+    @ConditionalOnProperty(prefix = "io.muenchendigital.digiwf.s3.client", name = "securityEnabled", havingValue = "true")
```

Comment:
You should mention this property in at least one properties-file to prevent having it undocumented.

Reply:
added it to the application-local.properties